x/poolmanager/protorev_test.go

```diff
@@ -2,49 +2,30 @@ package poolmanager_test
 
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/osmosis-labs/osmosis/osmoutils"
 )
 
-func (s *KeeperTestSuite) TestGetSetTakerFeeTrackerForStakersAndCommunityPool() {
+func (s *KeeperTestSuite) TestGetTakerFeeTrackerForStakersAndCommunityPool() {
```

Comment:
We remove Setter logic in this test. We now only have Get and Update, where update only increases and does not overwrite.